core/aggregator/api/api_domain.go

```diff
@@ -32,15 +32,15 @@ func CreateDomain(w http.ResponseWriter, r *http.Request) {
 
 		if err != nil {
 			fmt.Println(err)
-			utils.RespondWithError(w, http.StatusBadRequest, "Invalid Product ID")
+			utils.RespondWithError(w, http.StatusBadRequest, "Invalid Domain ID")
```

Comment:
Separe all this same change in another commit like `Fix error message of domain`